.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:jest/recommended',
+    'plugin:prettier/recommended',
+  ],
+  env: {
+    browser: true,
+    node: true,
+  },
+  parser: 'babel-eslint',

```

Comment:
Use the [Babel](https://babeljs.io/) parser which allows the use of keywords such as `import` and `export`